<p><img alt="caee logo" src="https://drive.google.com/uc?export=view&id=1bLjob5TsJBD3tWzNKzi6sbseyXsm6eb-" align="left" hspace="10px" vspace="0px"></p>
<p><img alt="iel logo" src="https://drive.google.com/uc?export=view&id=1uoCP8K9MyHTmU5nnpdlZoprhO8Bm5Yq5" align="right" hspace="10px" vspace="0px"></p>
<h1 align='center'>Homework 3</h1>

ARE 346P / CE 389H HVAC Design

Professor Dr. Zoltan Nagy

**Due: October 31st, 2020**

<h2>Instructions</h2>

Solve each of the following problems and show your work. Make sure to label each answer with the same label as the problem.

You will be graded on both the correct method and solution. Partial credit is awarded for use of the correct equation or method, correct values chosen from tables and charts when required, and the correct answer with appropriate units. 

This assignment includes *two* problems based on Chapter 11 from your textbook.

| PROBLEM | POINTS |
| --- | --- |
| 11.1 | 15 pts |
| 11.3 | 15 pts |
| **TOTAL** | **30 pts** |

## Resources

For this Homework, the following resources might be useful for you:

1. Psychrometric Chart
    1. [Online Interactive Chart](http://www.flycarpet.net/en/psyonline)
    2. [Downloadable Charts](http://www.handsdownsoftware.com)
1. Table A.4 from your textbook
3. Table A.5
3. Table A.6
3. Table A.1

# Problems

## 11.1

Two thousand cubic feet per minute of saturated moist air at 50$^\circ$F and a pressure of 14.696 psia is heated to 120$^\circ$F in a single-pass, cross-flow heat exchanger. Hot water enters at 180$^\circ$F at a volumetric flow rate of 30 gpm. The hot water is mixed, the air is unmixed. Determine the:

1. leaving-water temperature,
2. value of the correction factor F,
3. value of LMTD,
4. heat-exchanger effectiveness,
5. fluid capacity-rate ratio,
6. value of NTU, and
7. overall UA product

![11p1](https://drive.google.com/uc?export=view&id=1zwOVbVypIJr0As8exdetE77YD9ZzVMuc)

### 11.1.1

From an energy balance on the heat exchanger we get:

$$
\dot{m}_\text{air} c_\text{p,air} \left(t_1 - t_2\right) = \dot{m}_\text{water} c_\text{p,water} \left(t_A - t_B\right)
$$

We can rearrange to solve for the temperature at B (the leaving-water temperature), but before we do that we need to determine:

- mass flow rate of air
- mass flow rate of water
- specific heat of air

#### Mass flow rate of air

To find the mass flow rate of air, we have to use the volumetric flow rate given to us and the specific volume at the given conditions. Using the psychrometric chart, we identify the point at 50$^\circ$F and $\phi$ = 100%.

In [2]:
# defining given values
dot_V1 = 2000

# defining values from the psychrometric chart
v1 = 13

# calculating the mass flow rate of air
dot_m1 = dot_V1 / v1

print(f'Mass flow rate of air: {dot_m1}')

Mass flow rate of air: 153.84615384615384


#### Mass flow rate of water
We will use the same equation to determine the mass flow rate of water. We can get the specific volume from Table A.1E.

In [3]:
# defining given values (and converting)
dot_VA = 30 * 35.32 / 264.2

# defining values from the psychrometric chart
vA = 0.01651

# calculating the mass flow rate of air
dot_mA = dot_VA / vA

print(f'Mass flow rate of air: {dot_mA}')

Mass flow rate of air: 242.91932354900638


#### Specific heat of air

Using Eq. 7.23 from the textbook:

$$
c_p = c_\text{p,a} + W c_\text{p,w}
$$

These two new values can be determined from Tables A.5E and A.6E, respectively, and the humidity ratio, $W$, can be determined from the psychrometric chart.

In [6]:
# defining values from Tables
cp_a = 0.240 #A.5E
cp_w = 0.444 #A.6E

# defining values from the psychrometric chart
W1 = 0.0076

# calculating the specific heat of air
cp_air = cp_a + W1 * cp_w

print(f'Specific heat of air: {cp_air}')

Specific heat of air: 0.2433744


#### Exit temperature

Rearranging the equation from the beginning and solving, we get the value we are looking for.

In [11]:
# defining given values
tA = 180
t1 = 50
t2 = 120

# defining assumed values
cp_water = 1

# calculating the temperature at B
tB = tA + (dot_m1 * cp_air * (t1 - t2))/(dot_mA * cp_water)

print(f'Exit temperature: {tB}')

Exit temperature: 169.21059453553795


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{T}_\text{B} = 169.2^\circ \text{F} $
</div>

### 11.1.2

To determine the correction factor, $F$, we must use the correct figure for a single-pass, cross flow heat exchanger with one fluid mixed and the other unmixed. To do so, we use Figure 7.15 from the textbook. We first need to calculate the two dimensionless parameters $R$ and $P$.

In [12]:
# calculating R and P
R = (tA - tB)/(t2 - t1)
P = (t2 - t1)/(tA - t1)

print(f'R: {R}\nP: {P}')

R: 0.15413436377802928
P: 0.5384615384615384


Now we use these two values to enter figure 7.15 to determine $F$.

<div class="alert alert-block alert-success">
<b>Solution:</b> $F = 0.99$
</div>

### 11.1.3

To solve this problem, we simply apply Eq. 11.4 from the textbook with information we have already determined:

$$
\Delta t_\text{m,cf} = \frac{\left(t_2 - t_1\right)(R-1)}{\ln \left(\frac{1 - R}{1 - RP}\right)}
$$

where the logarithmic mean temperature difference (LMTD) for **pure counterflow** is $\Delta t_\text{m,cf}$.

In [19]:
# importing library to use natural logarithm
import math

# calculating LMTD for counterflow
lmtd_counter = ( (t2 - t1) * (R - 1) )/ ( math.log((1 - P)/(1 - R * P)) )

print(f'LMTD (pure counterflow): {lmtd_counter}')

LMTD (pure counterflow): 86.24404243657759


Now we use the correction factor to "convert" from pure counterflow to crossflow.

In [20]:
# defininng previously determined values
F = 0.99

# calculating LMTD for crossflow
lmtd_cross = F * lmtd_counter

print(f'LMTD (crossflow): {lmtd_cross}')

LMTD (crossflow): 85.38160201221181


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{LMTD} = 85.4^\circ$F
</div>

### 11.1.4

When determining the effectiveness, we have to determine which fluid has the larger capacity rate.

In [24]:
# calculating capacities
c_air = dot_m1 * cp_air
c_water = dot_mA * cp_water

print(f'Capacity (air): {c_air}\nCapacity (water): {c_water}')

Capacity (air): 37.44221538461538
Capacity (water): 242.91932354900638


Since the capacity rate of air is less than water, $c_\text{min} = c_\text{air} = c_\text{cold}$. This relationship means that the temperature of the air will change more than the temperature of the water. Therefore by Eq. 11.19, we can determine the efficiency, $\epsilon$.

<div class="alert alert-block alert-success">
<b>Solution:</b> $\epsilon = P = 0.54$
</div>

### 11.1.5

The fluid capacity ratio is given by Eq. 11.21:

$$
c_r = \frac{c_\text{cold}}{c_\text{hot}}
$$

In [26]:
# calculating fluid capacity ratio
c_r = c_air / c_water

print(f'Fluid capacity ratio: {c_r}')

Fluid capacity ratio: 0.15413436377802944


<div class="alert alert-block alert-success">
<b>Solution:</b> $c_r = 0.15$
</div>

### 11.1.6

We can determine the equation for the number of transfer units (NTU), by using Table 11.2 from your textbook. We have two options for crossflow heat exchangers and, in our case, $c_\text{min}$ is unmixed so we have the first case:

$$
\epsilon = \frac{1}{c_r} \left[1 - \exp\left(-c_r \left[1 - \exp\left(-\text{NTU}\right)\right]\right)\right]
$$

After careful rearranging, we get:

$$
\text{NTU} = - \ln \left[1 + \frac{1}{c_r} \ln\left(1 - \epsilon c_r\right)\right]
$$

In [28]:
# defining previously determined values
epsilon = P

# calculating NTU
NTU = -1 * math.log(1 + 1 / c_r * math.log(1 - epsilon * c_r))

print(f'NTU: {NTU}')

NTU: 0.8258223931468816


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{NTU} = 0.83$
</div>

### 11.1.7

The UA product is determined from rearranging Eq. 11.20 from your textbook:

$$
U_o A_o = c_\text{cold} \text{NTU}
$$

In [31]:
# calculating UA product
UA = c_air * NTU

print(f'UA: {UA}')

UA: 30.92061991364406


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{U}_o\text{A}_o = 30.92 \frac{\text{Btu}}{\text{min}^\circ F} = 1853 \frac{\text{Btu}}{\text{hr}^\circ F}$
</div>

## 11.3

Repeat Problem 11.1 if the heating medium is condensing steam at a pressure of 10 psig that enters as saturated vapor and leaves as saturated liquid:

1. leaving-water temperature,
2. value of the correction factor F,
3. value of LMTD,
4. heat-exchanger effectiveness,
5. fluid capacity-rate ratio,
6. value of NTU, and
7. overall UA product


![11p3](https://drive.google.com/uc?export=view&id=1DvvKdSFMe-w1k36n6fgDtLXIy6-G4B95)

### 11.3.1

The leaving temperature will be easier to calculate in this instance since we know we have saturated water at the outlet. We just have to use Table A.1E to determine the value.

<div class="alert alert-block alert-success">
<b>Solution:</b> $t_B = 239.38^\circ$F
</div>

### 11.3.2

The correction factor is simply 1 since one of the fluids is isothermal.

<div class="alert alert-block alert-success">
<b>Solution:</b> $F = 1$
</div>

### 11.3.3

Since the fluid temperature remains constant, we use Eq. 11.18 from the textbook to calculate the LMTD.

In [34]:
# defining previously determined values
tB = 239.38

# calculating LMTD
LMTD = (t2 - t1) / math.log((tB - t1) / (tB - t2))

print(f'LMTD: {LMTD}')

LMTD: 151.697748382583


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{LMTD} = 151.7^\circ$F
</div>

### 11.3.4

To calculate the efficiency, we are going to use the same formula we did before (11.19) since the hot fluid isn't changing temperature at all.

In [36]:
# calculating the efficiency
epsilon = (t2 - t1) / (tB - t1)

print(f'Efficiency: {epsilon}')

Efficiency: 0.3696272045622558


<div class="alert alert-block alert-success">
<b>Solution:</b> $\epsilon = 0.370$
</div>

### 11.3.5

Since the steam is undergoing a phase change and not changing in termperature, $c_r$ = 0 since $c_\text{max} = c_\text{hot} = \infty$

<div class="alert alert-block alert-success">
<b>Solution:</b> $c_r = 0$
</div>

### 11.3.6

Again, since one of our fluids is isothermal, we use Eq. 11.25 from the textbook and rearrange for NTU.

$$
\text{NTU} = -\ln \left(1 - \epsilon\right)
$$

In [37]:
# calculating NTU
NTU = -1 * math.log(1 - epsilon)

print(f'NTU: {NTU}')

NTU: 0.4614438958148502


<div class="alert alert-block alert-success">
<b>Solution:</b> NTU = 0.46
</div>

### 11.3.7

We will calculate the UA product the same way using values previously determined in 11.1.

In [41]:
# calculating UA product
UA = c_air * NTU

print(f'UA: {UA}')

UA: 17.27748173501564


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{U}_o\text{A}_o = 17.28 \frac{\text{Btu}}{\text{min}^\circ F} = 1037 \frac{\text{Btu}}{\text{hr}^\circ F}$
</div>